File: ios/Plugin/DynatracePlugin.swift
import Foundation
import Capacitor

/**
 * Please read the Capacitor iOS Plugin Development Guide
 * here: https://capacitorjs.com/docs/plugins/ios
 */
@objc(DynatracePlugin)
public class DynatracePlugin: CAPPlugin {
    private let implementation = Dynatrace()
}

File: android/src/main/java/com/okode/dynatrace/DynatracePlugin.java
package com.okode.dynatrace;

import com.getcapacitor.Plugin;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "Dynatrace")
public class DynatracePlugin extends Plugin {

    private Dynatrace implementation = new Dynatrace();
}
